cpp/src/rolling/rolling.cu

```diff
@@ -766,6 +777,9 @@ struct rolling_window_launcher {
              rmm::mr::device_memory_resource* mr,
              cudaStream_t stream)
   {
+    CUDF_EXPECTS(default_outputs.is_empty() || op == aggregation::LEAD || op == aggregation::LAG,
```

Comment:
`op` can't be `LEAD` or `LAG` here, yes?
```suggestion
    CUDF_EXPECTS(default_outputs.is_empty() ,
```

Reply:
Yikes! Great catch. I'll remove that redundant check.

Reply:
Just fixed that, thank you.